src/language-js/print/call-arguments.js

```diff
@@ -74,17 +76,15 @@ function printCallArguments(path, options, print) {
       arg.type !== "ArrowFunctionExpression" ||
       !arg.body ||
       arg.body.type !== "BlockStatement" ||
-      !arg.params ||
-      arg.params.length < 1
+      getFunctionParameters(arg).length === 0
     ) {
       return false;
     }
 
     let shouldBreak = false;
-    argPath.each((paramPath) => {
-      const printed = concat([print(paramPath)]);
-      shouldBreak = shouldBreak || willBreak(printed);
-    }, "params");
+    iterateFunctionParametersPath(argPath, (paramterPath) => {
```

Comment:
`paramterPath` → `parameterPath`. Why didn't cspell catch this?

Reply:
https://github.com/streetsidesoftware/cspell-dicts/blob/ea86559a6f97853a8af6218ae8c0b2b1c7c36f69/packages/node/node.txt#L1347